Paws4Me/Paws4Me/Utilities/URLSession+Extension.swift

```diff
@@ -14,14 +14,14 @@ extension URLSession {
         case invalidUrl
         case invalidData
     }
-    func makeRequest<Generic: Codable>(url: URLRequest?,
+    func makeRequest<Generic: Codable>(url: URL?,
                                        model: Generic.Type,
                                        completion: @escaping (Result<Generic, Error>) -> Void) {
         guard let endpointUrl = url else {
             completion(.failure(CustomError.invalidUrl))
             return
         }
-        let apiTask = self.dataTask(with: endpointUrl as URLRequest) { data, _, error in
+        let apiTask = self.dataTask(with: endpointUrl as URL) { data, _, error in
```

Comment:
No need for this cast -please prioritise getting this branch up to date with develop.

Reply:
This has been done in the other PR and it is now up to date with develop 👍🏻